src/integratedServer/worker.ts

```
@@ -0,0 +1,163 @@
+    message: string,
+    suggestCommand?: string,
+    isError?: boolean,
+  }
+}
+
+const postMessage = <T extends keyof BackEvents>(type: T, data?: BackEvents[T], ...args) => {
+  try {
+    globalThis.postMessage({ type, data }, ...args)
+  } catch (err) {
+    // eslint-disable-next-line no-debugger
+    debugger
```

Biome (1.9.4)

[error] This is an unexpected use of the debugger statement.

Unsafe fix: Remove debugger statement


(lint/suspicious/noDebugger)